src/VisualStudio/Core/Def/LanguageService/AbstractPackage.cs

```diff
@@ -27,10 +27,6 @@ internal IComponentModel ComponentModel
 
     protected override async Task InitializeAsync(CancellationToken cancellationToken, IProgress<ServiceProgressData> progress)
     {
-        await base.InitializeAsync(cancellationToken, progress).ConfigureAwait(true);
```

Comment:
is it ok/intentional to not  cal the base here?  i'd actually like a comment saying that thsi is intentional, so it doesn't come backk in the future.

Reply:
We could doc it if you want, but no package in VS calls InitializeAsync anymore. [I've documented this](https://devdiv.visualstudio.com/DevDiv/_git/VS?path=/src/env/shell/PackageFramework/Current/Shell/AsyncPackage.cs&version=GBmain&line=85&lineEnd=85&lineStartColumn=1&lineEndColumn=135&lineStyle=plain&_a=contents).

Reply:
doc'ing would def be nice.  just so we have smething to state that this is very intentional as opposed to accidental.